core/base/src/test/java/org/eclipse/dataspaceconnector/core/CoreServicesExtensionTest.java

```diff
@@ -14,40 +14,51 @@
 
 package org.eclipse.dataspaceconnector.core;
 
+import org.eclipse.dataspaceconnector.core.security.DefaultPrivateKeyParser;
 import org.eclipse.dataspaceconnector.policy.model.PolicyRegistrationTypes;
 import org.eclipse.dataspaceconnector.spi.security.PrivateKeyResolver;
 import org.eclipse.dataspaceconnector.spi.system.ServiceExtensionContext;
 import org.eclipse.dataspaceconnector.spi.types.TypeManager;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyInt;
 import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 class CoreServicesExtensionTest {
     private CoreServicesExtension extension;
     private ServiceExtensionContext context;
     private TypeManager typeManager;
+    private PrivateKeyResolver privateKeyResolverMock;
 
     @Test
     void verifyPolicyTypesAreRegistered() {
         extension.initialize(context);
         PolicyRegistrationTypes.TYPES.forEach(t -> verify(typeManager).registerTypes(t));
     }
 
+    @Test
+    void verifyDefaultPrivateKeyParserIsRegistered() {
+        extension.initialize(context);
+        verify(privateKeyResolverMock, times(1)).addParser(any(DefaultPrivateKeyParser.class));
```

Comment:
```suggestion
        verify(privateKeyResolverMock).addParser(any(DefaultPrivateKeyParser.class));
```

See JavaDoc for `org.mockito.Mockito#verify(T)`